src/main/java/meteordevelopment/meteorclient/mixin/ClientPlayerEntityMixin.java

```diff
@@ -73,7 +73,8 @@ private void onPushOutOfBlocks(double x, double d, CallbackInfo info) {
 
     @Inject(method = "damage", at = @At("HEAD"))
     private void onDamage(DamageSource source, float amount, CallbackInfoReturnable<Boolean> info) {
-        if (Utils.canUpdate() && world.isClient && canTakeDamage()) MeteorClient.EVENT_BUS.post(DamageEvent.get(this, source, amount));
+        if (Utils.canUpdate() && getWorld().isClient && canTakeDamage())
```

Comment:
Same thing as prior comment, please do not do cleanups that are un-related to 1.20 -- store in seperate PR